index.js

```diff
@@ -7,25 +8,34 @@ const untildify = require('untildify');
 
 const bplist = pify(bplistParser);
 const settings = path.join(userHome, '/Library/Preferences/com.runningwithcrayons.Alfred-Preferences-3.plist');
+const prefsJsonPath = path.join(userHome, '/Library/Application Support/Alfred/prefs.json');
 
 module.exports = async () => {
 	let data;
+	let errorMessage;
+
+	try {
+		const prefsPath = JSON.parse(fs.readFileSync(prefsJsonPath)).current;
+
+		return {path: prefsPath};
```

Comment:
Spread over multiple lines
```suggestion
		return {
			path: prefsPath
		};
```